client/galaxy/scripts/mvc/workflow/workflow-view-terminals.js

```diff
@@ -253,10 +264,10 @@ var OutputTerminalView = BaseOutputTerminalView.extend({
     terminalMappingClass: Terminals.TerminalMapping,
     terminalMappingViewClass: TerminalMappingView,
     terminalForOutput: function(output) {
-        var type = output.extensions;
-        var terminal = new Terminals.OutputTerminal({
+        var type = output.type;
+        var terminal = new Terminals.OutputParameterTerminal({
             element: this.el,
-            datatypes: type
```

Comment:
Should these changes be reverted - given the class below?

Reply:
Oh yeah, sure.

Reply:
Going to remove this via rebase on this commit and force push to your branch - hope that is okay.